### PR TITLE
mgmt, regen trafficmanager

### PR DESCRIPTION
**Pull request by weidongxu-microsoft**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Comment by weidongxu-microsoft** [comment]
1. test the latest codegen (there is some difference of API in fluent package)
2. regen for samples

File: sdk/resourcemanager/azure-resourcemanager-trafficmanager/src/main/java/com/azure/resourcemanager/trafficmanager/fluent/models/EndpointInner.java [hunk 1]
```diff
@@ -5,123 +5,35 @@
...
-     */
-    @JsonProperty(value = "properties.customHeaders")
-    private List<EndpointPropertiesCustomHeadersItem> customHeaders;
-
-    /*
-     * Fully qualified resource Id for the resource. Ex -
-     * /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/trafficManagerProfiles/{resourceName}
-     */
-    @JsonProperty(value = "id")
-    private String id;
```
Comment (XiaofeiCao) [inline sdk/resourcemanager/azure-resourcemanager-trafficmanager/src/main/java/com/azure/resourcemanager/trafficmanager/fluent/models/EndpointInner.java:112]: Removal of `id`, `name` and `type` seems not correct. 

Did you merge the latest autorest.java main? I had an PR merged earlier to address this issue. If so, I have to take another look...

Comment (XiaofeiCao) [reply sdk/resourcemanager/azure-resourcemanager-trafficmanager/src/main/java/com/azure/resourcemanager/trafficmanager/fluent/models/EndpointInner.java:112]: OK, I saw the fix, never mind.

Comment (weidongxu-microsoft) [reply sdk/resourcemanager/azure-resourcemanager-trafficmanager/src/main/java/com/azure/resourcemanager/trafficmanager/fluent/models/EndpointInner.java:112]: Yeah, the PR wasn't merged with the ProxyResource fix.

**Review by haolingdong-msft — APPROVED** [review]

**Comment by weidongxu-microsoft** [comment]
live tests passed

**Pull request merged**